docs/streams/unsubscribe-streams.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 80
+title: Unsubscribe from Streams
+---
+
+This page explains how you can unsubscribe from streams in Macrometa.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="operating-systems">
+<TabItem value="api-py" label="API - Python">
+
+Use our interactive API Reference with code generation in 18 programming languages to [Delete All Subscriptions](https://macrometa.com/docs/api#/operations/DeleteAllSubscriptions) from a stream or [Remove a Subscription](https://macrometa.com/docs/api#/operations/DeleteSubscription).
+
+```py
+import requests
+
+# Constants
+URL = "api-gdn.paas.macrometa.io"
+HTTP_URL = f"https://{URL}"
+FABRIC = "_system"
+STREAM_NAME = "streamQuickstart"
+IS_GLOBAL = True # For a global stream pass global=true and global=false for local stream
+API_KEY = "XXXXX" # Use your apikey here
+AUTH_TOKEN = f"apikey {API_KEY}" # apikey keyword needs to be appended
+
+session = requests.session()
+session.headers.update({"content-type": 'application/json'})
+session.headers.update({"authorization": AUTH_TOKEN})
+
+# Delete subscription
+url = f"{HTTP_URL}/_fabric/{FABRIC}/_api/streams/{stream_type}s.{STREAM_NAME}/subscriptions/{CONSUMER_NAME}?global=true"
+resp = session.delete(url)
+print("Subscription deleted: ", resp.text)
+```
+
+</TabItem>
+<TabItem value="api-js" label="API - JS">
+
+Use our interactive API Reference with code generation in 18 programming languages to [Delete All Subscriptions](https://macrometa.com/docs/api#/operations/DeleteAllSubscriptions) from a stream or [Remove a Subscription](https://macrometa.com/docs/api#/operations/DeleteSubscription).
```

Comment:
@diana-macrometa I think there is a major misunderstanding here... It is not deleting all subscription from a stream... Unsubscribe API operation means removing the given subscription from all streams either global or local) which is defined by a "global" parameter which we give as input to the API.. I think it would be best if you login to smoke1 or smoke3 and check the API references from the UI that should clear the picture for you..

And then the second one delete subscription is as it is like just delete a given subscription from a given stream
But the deleteallsubscriptions thing is wrong.. plus our apidocs on docs repo seems not updated from a while.. (like I don't see the "global" parameter in unsubscribe method of api ref in docs but it is present any GDN newer versions

Reply:
Theoretically, the API docs are built from the same Swagger .json file. I don't think they update except when a release goes out.

Reply:
I see that this file:
https://github.com/Macrometacorp/docs/tree/master/static/swagger
hasn't been updated in 2 months

Reply:
LOL. Bold of you to assume I know what it meant!